akommons/src/main/java/com/dewarder/akommons/view/View.kt

```diff
@@ -60,34 +62,35 @@ inline fun <V : View> View.findView(
 ): V = findViewById<V>(id).apply(init)
 
 
-fun View.setAllPadding(padding: Int) {
-    setPadding(padding, padding, padding, padding)
-}
+fun View.setAllPadding(padding: Int) = setPadding(padding, padding, padding, padding)
```

Comment:
Please, revert back all functions to use block body instead of expression one. The purpose of block body is to prevent accidentally changing of return type and omit explicit `Unit` as return type.